### DISCUSSION TITLE
[Feature]: Option to use <a> instead of <button> for tab

### DISCUSSION THREAD

**Discussion by RhysSullivan** [tremorlabs/tremor-npm#685] [Ideas]
### What problem does this feature solve?

I'd like to be able to use an <a> tag instead of a button for the Tab primitive it'd allow for the state to be persisted in the url instead on the client

Love using tremor, thanks for making this library

### What does the proposed API look like?

<Tab href='/tab2'... />

**Comment by severinlandolt** [comment]
Hey everyone! We're curious about the community's thoughts on this proposal. It's a solid concept with numerous potential applications. However, we utilize Headless UI, which defaults to a button. Eager to hear your potential solutions or past experiences with this!

**Comment by christopherkindl** [comment]
We got this request a lot recently. Also, including ourselves experienced the need to use href instead of a button when creating new blocks. Hence, we have submitted this as an issue. Thanks @RhysSullivan for this!

Reference: https://github.com/tremorlabs/tremor/issues/814
